test_unstructured_ingest/expected-structured-output/azure-blob-storage/IRS-form-1987.png.json

```diff
@@ -267,7 +267,7 @@
       "filetype": "image/png",
       "page_number": 1
     },
-    "text": "We ask for this information to carry out the\nInternal Revenue laws of the United States. We\nneed it to ensure that taxpayers are complying\nwith these laws an¢ to allow us to figure and\n\ncollect the right amount of tax. You are required\nto give us this information,\n"
+    "text": "We ask for this information to carry out the Internal Revenue laws of the United States. We need it to ensure that taxpayers are complying with these laws an¢ to allow us to figure and collect the right amount of tax. You are required to give us this information,"
```

Comment:
it looks like we're overly aggresive stripping out newlines:
![image](https://github.com/Unstructured-IO/unstructured/assets/28578599/0897fd9a-2700-4263-aa6b-51160968dc82)

Reply:
although some use cases may prefer this, hmm